node/src/sub_lib/blockchain_bridge.rs

```diff
@@ -22,6 +23,7 @@ pub struct BlockchainBridgeConfig {
 pub struct BlockchainBridgeSubs {
     pub bind: Recipient<BindMessage>,
     pub report_accounts_payable: Recipient<ReportAccountsPayable>,
+    pub request_available_balances_for_payables: Recipient<RequestAvailableBalancesForPayables>,
```

Comment:
In case you change the name of this message, please change the name of this field too.

Reply:
See the first answer to your comment touching this.

Reply:
Ok, let's do that.